packages/cli/test/dev/utils.js

```diff
@@ -430,7 +433,7 @@ function testFixtureStdio(
           );
         }
 
-        if (stderr.includes('Command failed') || stderr.includes('Error!')) {
+        if (stderr.includes('Command failed')) {
```

Comment:
This `[vercel dev] Middleware with error at init` test relies on the `vc dev` not having `Error!` in the `stderr` output, which it now does.

`vc dev` and real deployments do still have the same behavior, but `testFixtureStdio` is being a bit aggressive in what counts as a failure. I removed this check for `Error!` and it now passes.

I wanted to make sure this check wasn't being relied on for other tests. So, I ran a full set of tests on just removing this check, which passed: https://github.com/vercel/vercel/pull/8293